enterprise/frontend/src/metabase-enterprise/audit_app/pages/AuditQuestionDetail.jsx

```diff
@@ -23,7 +23,7 @@ const AuditQuestionDetail = ({ params, ...props }: Props) => {
     <AuditContent
       {...props}
       title={<EntityName entityType="questions" entityId={questionId} />}
-      subtitle={<OpenInMetabase to={Urls.question(questionId)} />}
+      subtitle={<OpenInMetabase to={Urls.question({ id: questionId })} />}
```

Comment:
Thank you! I completely forgot we have frontend code inside `enterprise` while implementing URL slugs 🤦 

Got the same question as for the dashboard URL. If it's an easy thing, it'd be awesome if we can pass the whole question object or its name to the `Urls.question`, so the function can generate a slug

Reply:
No worries, we all missed that during the review. At least we fixed it before the release